securesystemslib/rsa_keys.py

```diff
@@ -473,9 +511,18 @@ def verify_rsa_signature(signature, signature_scheme, public_key, data):
     # hashing algorithm.
     try:
       if signature_scheme.startswith('rsassa-pss'):
+        if salt_length == SaltLength.PSSSaltLengthAuto:
+          salt_length = padding.PSS.MAX_LENGTH
+        elif salt_length == SaltLength.PSSSaltLengthEqualsHash:
+          salt_length = digest_obj.algorithm.digest_size
+        else:
+          raise securesystemslib.exceptions.UnsupportedAlgorithmError('Unsupported'
+            ' salt length is specified: ' + repr(salt_length))
```

Comment:
Perhaps we could factor this logic out into a helper, as we use it here in verify and earlier in create.